build/build.xml

```diff
@@ -387,6 +387,7 @@
 		<build-addon name="sniTerminator" />
 		<build-addon name="soap" />
 		<build-addon name="sse" />
+		<build-addon name="tlsDebug" />
```

Comment:
The name should be all lower case (to match the directory, otherwise it leads to an error in case sensitive file systems).